lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -133,14 +132,15 @@ SessionState::SessionState(
     const StoredSessionState& marshaled, StaticRuleStore& rule_store)
     : imsi_(marshaled.imsi),
       session_id_(marshaled.session_id),
-      local_teid_(marshaled.local_teid),
       request_number_(marshaled.request_number),
       curr_state_(marshaled.fsm_state),
       config_(marshaled.config),
       pdp_start_time_(marshaled.pdp_start_time),
       pdp_end_time_(marshaled.pdp_end_time),
       // 5G session version handlimg
       current_version_(marshaled.current_version),
+      // SMF-UPF version missmatch, max re-sending throttle count
```

Comment:
typo `missmatch`

Reply:
max re-sending throttle count -> retransmission counter ?

Reply:
Yes, it is retransmission counter. Corrected it.